pre_push.py

```diff
@@ -24,11 +27,14 @@ def do_process(args, shell=False):
     except Exception as exc:
         sys.stderr.write(str(exc) + "\n")
         sys.exit(1)
-    return True
+    else:
+        return True
 
 
 def main():
-    """Entry point to pre_push.py."""
+    """Entry point to pre_push.py.
+    Returns a statuscode of 0 if everything ran correctly.
```

Comment:
I believe doc formatting recommends there be a blank line before adding subsequent comments from the short message. I believe the tool `pydocstyle` should point out said message.

Reply:
My pydocstyle did not point this out. I still fixed it however.